legal-api/src/legal_api/resources/business/business_filings.py

```diff
@@ -479,7 +479,7 @@ def _get_filing_types(filing_json: dict):
         return filing_types
 
     @staticmethod
-    def _create_invoice(business: Business,
+    def _create_invoice(business: Business,  # pylint: disable=too-many-locals
```

Comment:
I wanted to keep with the pattern within this function, so I disabled this pylint error.